### PR TITLE
[r2.5] Update of oneDNN version for mkl_aarch64 build target

### PR DESCRIPTION
**Pull request by alenik01**
This PR noticeably improves the performance of CNNs on AArch64 (`--config=mkl_aarch64`). oneDNN version upgrade follows the PR #48035

### CODE REVIEW

**Comment by alenik01** [comment]
@penpornk @agramesh1 please have a look.

**Comment by penpornk** [comment]
@alenik01 Oh, I thought you are going to upgrade this in master and not 2.5. 

For 2.5, I'm not sure if we can cherry-pick this, since cherry-picks are for security / bug fixes. Upgrading from a v2.1 release to v2.2 release might count more towards adding features rather fixes. (The upgrade we did [here](https://github.com/tensorflow/tensorflow/pull/48345) was from a random commit after `v2.2-rc` to `v2.2` for more stability and fixes.)

@mihaimaruseac What do you think?

File: tensorflow/workspace2.bzl [hunk 1]
```diff
@@ -191,11 +191,11 @@ def _tf_repositories():
...
         name = "mkl_dnn_acl_compatible",
         build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
-        sha256 = "5f7fd92e2d0bf83580656695d4404e2cd1390ecad36496fd8ba10b5adc905f70",
-        strip_prefix = "oneDNN-2.1",
+        sha256 = "4d655c0751ee6439584ef5e3d465953fe0c2f4ee2700bc02699bdc1d1572af0d",
+        strip_prefix = "oneDNN-2.2",
         urls = [
-            "https://storage.googleapis.com/mirror.tensorflow.org/github.com/oneapi-src/oneDNN/archive/v2.1.tar.gz",
-            "https://github.com/oneapi-src/oneDNN/archive/v2.1.tar.gz",
+            "https://storage.googleapis.com/mirror.tensorflow.org/https://github.com/oneapi-src/oneDNN/archive/v2.2.tar.gz",
```
Comment (penpornk) [inline tensorflow/workspace2.bzl:197]: Typo:
```suggestion
            "https://storage.googleapis.com/mirror.tensorflow.org/github.com/oneapi-src/oneDNN/archive/v2.2.tar.gz",
```

**Review by penpornk — CHANGES_REQUESTED** [review]

**Comment by alenik01** [comment]
@penpornk if it is possible to cherry-pick this for v2.5 release, then it would be nice (the problem was that oneDNN v2.2 had been release after TF feature freeze deadline). If not, then this PR can be merged in the master.

**Comment by penpornk** [comment]
@alenik01 Could you please share how much performance improvements you observe from this update? Is this update for performance improvements alone or is it also fixing some issues?

**Comment by penpornk** [comment]
@alenik01 Also, could you please create another PR for the `master` branch? We usually only merge a cherry-pick after seeing the same changes successfully merged (and passed nightly tests) in `master`. 

I will check with the team if this PR is eligible for cherry-picking when I hear back about the speedups and whether this fixes any issues as well. Thank you!

**Comment by alenik01** [comment]
@penpornk
> Could you please share how much performance improvements you observe from this update?

The speedup is noticeable, x1.7 (was measure for ResNet50 inference).

> Is this update for performance improvements alone or is it also fixing some issues?

Performance alone.

> Also, could you please create another PR for the master branch?

OK, I will create one.

> since cherry-picks are for security / bug fixes

Many thanks for clarifying, now I understand that the version update for `mkl_dnn_v1` was one-off case.

**Comment by nSircombe** [comment]
@mihaimaruseac - this is the PR we briefly discussed at the SIG build meeting the other week.

**Comment by alenik01** [comment]
@penpornk
>Also, could you please create another PR for the master branch? 

Here it is: https://github.com/tensorflow/tensorflow/pull/48574

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**